engine/Shopware/Components/Translation.php

```diff
@@ -391,6 +397,327 @@ public function write($language, $type, $key = 1, $data = null, $merge = false)
     }
 
     /**
+     * Helper method to translate orders.
+     * Method extracts all documents, payment methods and dispatch methods of the order
+     * and translates them.
+     *
+     * @param array $orders
+     * @param int   $context
```

Comment:
Please don't indent the parameter names, because keeping the indentation up to date in future changes might clutter the diff (since all rows must be updated).

Reply:
As discussed personally, the indentation was created by shopware's git hook.

Reply:
As discussed personally, the indentation was created by shopware's git hook.